modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -394,7 +395,8 @@ private PartialNode startPartialNode(
                 new VaultStaleIds(vault),
                 clusterIdService,
                 workerRegistry,
-                failureProcessor
+                failureProcessor,
+                defaultChannelTypeRegistry()
```

Comment:
How about creating an overload of `createClusterService()` with the old set of parameters that would just do `defaultChannelTypeRegistry()` itself, to avoid changing all these tests?

Reply:
I would leave it as is, since other arguments can be removed there, I suggest not doing it in this PR.